ground/src/test/java/com/google/android/ground/model/geometry/GeometryTest.kt

```diff
@@ -74,7 +74,7 @@ class GeometryTest {
   @Test
   fun testPointArea() {
     val point = point(x, y)
-    assertThat(POINT_AREA).isEqualTo(point.area)
+    assertThat(ZERO_AREA).isEqualTo(point.area)
```

Comment:
Can we change the order of actual result and expected result? Also, let's hardcode the value 0 instead of creating a variable for simplicity. Something like;

```
assertThat(point.area).isEqualTo(0.0)
```

Reply:
done